config/rbac/role.yaml

```diff
@@ -4,6 +4,12 @@ kind: ClusterRole
 metadata:
   name: manager-role
 rules:
+- apiGroups:
+  - '*'
+  resources:
+  - '*'
+  verbs:
+  - '*'
```

Comment:
This is equivalent to letting operator-controller watch all the resources in cluster. This was added to rukpak to allow watching of any bundle resources. So keeping it the same for now.